core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -620,8 +620,8 @@ else if (header == params.getBip32HeaderP2WPKHpub() || header == params.getBip32
     }
 
     /**
-     * Creates a new, empty wallet with a randomly chosen seed and no transactions. Make sure to provide for sufficient
-     * backup! Any keys will be derived from the seed. If you want to restore a wallet from disk instead, see
+     * Creates a new, empty wallet with no transactions. Make sure to provide for sufficient backup!
+     * Any keys will be derived from the seed. If you want to restore a wallet from disk instead, see
```

Comment:
But shouldn't this line go away as well? (the hint about restoring from disk maybe reworded)

Reply:
Yes, give me a few minutes and I'll force-push an update.

Reply:
Done.